functions.js

```diff
@@ -103,6 +103,9 @@ function loadTodos() {
   $.ajax({
           url: API_URL + "/todos",
           crossDomain: true,
+          method: "GET",
```

Comment:
add the data type to get a valid jsons response, otherwise it will be used as string (that's why you had the `undefined` elements)